media_daemon/99-susi-usb-drive.rules

```diff
@@ -1,4 +1,4 @@
 # Ref: https://unix.stackexchange.com/questions/229987/udev-rule-to-match-any-usb-storage-device
 
-ACTION=="add", KERNEL=="sd?", SUBSYSTEM=="block", ENV{ID_BUS}=="usb", RUN="/home/pi/SUSI.AI/susi_linux/media_daemon/autostart.sh"
+ACTION=="add", KERNEL=="sd?", SUBSYSTEM=="block", ENV{ID_BUS}=="usb", RUN="/usr/bin/sudo -u pi /home/pi/SUSI.AI/susi_linux/media_daemon/autostart.sh"
```

Comment:
Don't call `sudo` in udev rule. Udev allow to switch to a specific user to run the command. Read doc and fix.